### PR TITLE
[QNEBE-640] Readthedoc fix

### PR DESCRIPTION
**Pull request by QFer**
* Readthedoc is generated again without blocking errors

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -5,8 +5,8 @@
 SPHINXOPTS    =
 SPHINXBUILD   = sphinx-build
 SOURCEDIR     = .
-BUILDDIR      = build
-PYTHON3       = python3
+BUILDDIR      = _build
```
Comment (cmvanb) [inline docs/Makefile:8]: What is the reason for prefixing the underscore to this directory?

**Review by cmvanb — COMMENTED** [review]
Tested and working with python 3.8.16.

Curious why the build directory changed? The underscored directory name is not gitignored, so it will pollute the git index. Can we revert it to `build`?

File: docs/Makefile (hunk 1 above)
Comment (QFer) [reply docs/Makefile:8]: Few reasons:
1. Build directory is configured in docs/make.bat as _build (inconsistent)
2. _build is used in other projects
3. _build will appear on the top of the directory list in IDE

I agree the .gitignore should be changed for this. Which I will do.

**Review by cmvanb — APPROVED** [review]
:+1:

**Review by cmvanb — APPROVED** [review]
(more approval!)

**Pull request merged**